Alt Text and Link Edits - Pt. 4 (iOS) (#3342)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/ios/content_cards/data_model.md

{% endtab %}
{% endtabs %}

If you want to change the card data after it's been sent by Braze, we recommend storing a deep copy of the card data locally, updating the data, and displaying it yourself. The cards are accessible via [`ABKContentCardsController`](https://appboy.github.io/appboy-ios-sdk/docs/interface_a_b_k_content_cards_controller.html).

## Content Card model

> Braze offers three Content Card types: banner, captioned image, and classic. Each type inherits common properties from a base `ABKContentCard` class, plus has additional properties as described below.

Reviewer: ```suggestion
Braze offers three Content Card types: banner, captioned image, and classic. Each type inherits common properties from a base `ABKContentCard` class and has the following additional properties.
```

|Property|Description|
|---|---|
| `image` | (Optional) This property is the URL of the card's image.|
| `title` | The title text for the card. |
| `cardDescription` | The body text for the card. |

> | `domain` | The link text for the property URL, like @"blog.braze.com". It can be displayed on the card's UI to indicate the action/direction of clicking on the card. |

Reviewer: ```suggestion
| `domain` | The link text for the property URL, like @"blog.braze.com". It can be displayed on the card's UI to indicate the action and direction of clicking on the card. |
```
